src/error.rs

```diff
@@ -98,6 +97,43 @@ pub struct Transport {
     response: Option<Response>,
 }
 
+/// Extensions to [`Error`].
+pub trait ErrorExt {
```

Comment:
Maybe `ResultExt` is more accurate

```suggestion
/// Extensions to [`Result<Response, Error>`].
pub trait ResultExt {
```

Reply:
true true